packages/app-spec-squad-chess/src/Matchmaker.js

```diff
@@ -265,9 +262,33 @@ const handleSendAnswer = (event) => {
   m.redraw()
 }
 
+export const sendMessage = (message) => {
```

Comment:
Is there a reason this is in the "handlers" section? Can we move it into the main functions section?

Reply:
Ah sure that would be fine. I moved it here so I could see it next to it's handler as I worked on them both so I didn't have to jump back and forth keeping them in sync with each other.

Reply:
Oh, woops, this is the mithril app -- I thought this was the send message in the matchmaking client when I wrote this. This is actually the only non-component non-handler function in this file, I think, so it's fine. Let's merge!